### PR TITLE
UI pt2

### PR DESCRIPTION
**Pull request by muskaanshah**
UI created for History page, authentication pages, modals
Please note: Functionality hasn't been added for any of these yet

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]
Code Looks Clean
UI looks Clean
ClassNames are self explanatory

File: src/components/Drawer/Drawer.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ function Drawer() {
 				Explore
 			</NavLink>
 			<NavLink
-				to="/"
+				to="/playlist"
 				className={({ isActive }) =>
 					isActive ? "active-link drawer-link" : "drawer-link"
 				}
```
Comment (JowelTisso) [inline src/components/Drawer/Drawer.jsx:29]: Good approach with active link in className

File: src/components/videocard.css [hunk 2]
```diff
@@ -61,8 +77,29 @@
...
 .videocard-modal-action:hover {
 	background-color: var(--grey-dark-opacity);
 }
+
+.video-card:hover,
+.video-card-horizontal:hover,
+.videocard-modal-action:hover {
+	cursor: pointer;
+}
+
```
Comment (JowelTisso) [inline src/components/videocard.css:93]: You can make a util class for pointer

**Review by JowelTisso — APPROVED** [review]
Rest Looks Good

File: src/components/videocard.css (hunk 2 above)
Comment (muskaanshah) [reply src/components/videocard.css:93]: It's on hover na, otherwise I would have made it

Comment (JowelTisso) [reply src/components/videocard.css:93]: Ok, got it

**Pull request merged**